pkg/minikube/cluster/cluster.go

```diff
@@ -72,7 +72,7 @@ func init() {
 
 // CacheISO downloads and caches ISO.
 func CacheISO(config cfg.MachineConfig) error {
-	if config.VMDriver != "none" {
+	if config.VMDriver == constants.DriverNone {
```

Comment:
This logic looks like it has been reversed ?

Reply:
ooops good catch. thanks for catching it

Reply:
Done